### ISSUE TITLE
grab the buildpacks from a list of forks

### ISSUE THREAD

**Issue by rvalyi** [fgrehm/devstep#103]
Hello,

May be people would feel more comfortable using Devstep if they can see easily how Devstep buildpack differ from their original versions.

What about having a list of buildpack URL's such as in Herokuish:
https://github.com/gliderlabs/herokuish/tree/master/buildpacks
and then clone them at the image build time?
Then you would fork these buildpacks, may be t would be easier for people to fix the buildpacks or contribute new ones. Just an idea...

**Comment by fgrehm** [comment]
Yeah, the other day I was actually wondering if devstep should leverage https://github.com/gliderlabs/herokuish somehow... Thoughts?

**Comment by rvalyi** [comment]
@fgrehm herokuish rocks. if you could use it that would be terrific indeed...
May be the hardest points would come from the user management and ensuring the unpreviliged user is the one we want in devstep. But I think herokuish supports it. At least I'm using it with the same USER from a build to another...
On our side, as for supporting our Odoo ERP, we walked more on deeply on the buildpack way at the point we got a usable one, so expect adoption of that on our side.
